text-ui-test/EXPECTED.TXT

```diff
@@ -46,16 +46,16 @@ Enter a command: There are 1 participants in dinner party! Here are your partici
 ------------------------
 Enter a command: Event added successfully
 ------------------------
-Enter a command: name
+Enter a command:
 Events successfully sorted by name!
```

Comment:
I think you have to remove the newline here (i.e. merge lines 49-50 into one line and do the same for lines 55-56).